templates/prometheus.yml.j2

```diff
@@ -17,8 +17,12 @@ remote_read:
   {{ prometheus_remote_read | to_nice_yaml(indent=2) | indent(2, False) }}
 {% endif %}
 
+{% if prometheus_rules_files != [] %}
```

Comment:
This check is not necessary, prometheus is smart enough to ignore this if there are no files with specified extension.